README.md

```diff
@@ -132,14 +133,14 @@ public class App {
         // Use `query()` to run a synchronous query.
         // Synchronous queries block the current thread until the query completes.
         // Accepts the query, expected result class, and a nullable set of query options.
-        QuerySuccess<Page<Product>> result = client.query(query, new PageOf<>(Product.class));
+        QuerySuccess<Page<Product>> result = client.query(query, pageOf(Product.class));
```

Comment:
When I tested, I still needed to use `new PageOf<>`. I might be missing something tho.

Reply:
hmmm, there's a static import that should work. it's just a helper method

Reply:
:doh: Yeah — Fixing the import fixed this. Please ignore.